### PR TITLE
Complete menu for Inequality

### PR DESCRIPTION
**Pull request by Morpheu5**
Supports maths and chemistry along with Boolean logic.

- [x] Fix CSS
  - Some CSS tweaking needed. Hard-coded tweaks are possible but I'd prefer a more general solution. I may have done something wrong in the first place so if anyone wants to step up to the challenge…
- [x] Test

### CODE REVIEW

**Comment by jsharkey13** [comment]
LGTM does flag this unused `examboard` variable, that looks like it was supposed to have been used but is not: https://github.com/isaacphysics/isaac-cs-app/pull/205/files#diff-8d41aa15c93c92a60e3c791d49fbd8f7R36 can it safely be removed? There's a `props.examboard` lower down the file: https://github.com/isaacphysics/isaac-cs-app/pull/205/files#diff-8d41aa15c93c92a60e3c791d49fbd8f7R77 now instead? @Morpheu5

**Comment by Morpheu5** [comment]
> LGTM does flag this unused `examboard` variable, that looks like it was supposed to have been used but is not: https://github.com/isaacphysics/isaac-cs-app/pull/205/files#diff-8d41aa15c93c92a60e3c791d49fbd8f7R36 can it safely be removed? There's a `props.examboard` lower down the file: https://github.com/isaacphysics/isaac-cs-app/pull/205/files#diff-8d41aa15c93c92a60e3c791d49fbd8f7R77 now instead? @Morpheu5

Fixed in 6e1e796c.

File: src/app/services/questions.ts [hunk 1]
```diff
@@ -5,13 +5,16 @@ import {IsaacNumericQuestion} from "../components/content/IsaacNumericQuestion";
...
+import {IsaacSymbolicQuestion} from "../components/content/IsaacSymbolicQuestion";
 
 // @ts-ignore as TypeScript is struggling to infer common type for questions
 export const QUESTION_TYPES = new Map([
     ["isaacMultiChoiceQuestion", IsaacMultiChoiceQuestion],
     ["isaacItemQuestion", IsaacItemQuestion],
     ["isaacParsonsQuestion", IsaacParsonsQuestion],
     ["isaacNumericQuestion", IsaacNumericQuestion],
+    ["isaacSymbolicQuestion", IsaacSymbolicQuestion],
+    //TODO: ["isaacSymbolicChemistryQuestion", isaacSymbolicChemistryQuestion],
```
[the remark this answers is not in the document]
Comment (Morpheu5) [reply src/app/services/questions.ts]: Fixed in f0d0157a

**Comment by jsharkey13** [comment]
- This seems to scroll back to the top of the page when you close the modal. Did it always do this on CS, because it's annoying and confusing on long pages?
 - It doesn't parse the existing `bestResponse` answer for symbolic questions, neither for text entry nor the Inequality modal itself. This doesn't appear to have worked in the other PR (#193) either.
 - It seems to place availableSymbols involving derivatives and deltas in only the integrals/derivative menu, whereas in the old world they appeared in both the symbol list and that menu. This will be confusing to abandon now!

**Comment by Morpheu5** [comment]
> * This seems to scroll back to the top of the page when you close the modal. Did it always do this on CS, because it's annoying and confusing on long pages?

No clue yet.

> * It doesn't parse the existing `bestResponse` answer for symbolic questions, neither for text entry nor the Inequality modal itself. This doesn't appear to have worked in the other PR (#193) either.

Done.

> * It seems to place availableSymbols involving derivatives and deltas in only the integrals/derivative menu, whereas in the old world they appeared in both the symbol list and that menu. This will be confusing to abandon now!

Done.

**Comment by bh412** [comment]
2nd and 3rd issue fixed

**Pull request merged**